extensions/sar/README.md

```diff
@@ -41,13 +41,28 @@ The exact metadata that would appear in a STAC Collection record will vary depen
 | sar:relative_orbit        | integer       | A relative orbit number associated with the acquisition. See below for details. |
 | sar:incidence_angle       | number        | The center incidence angle is the angle defined by the incident radar beam at the scene center and the vertical (normal) to the intercepting surface. Measured in degrees (0-90). |
 
-**sar:absolute_orbit** usually corresponds to the number of orbits elapsed since satellite launch (e.g. ALOS, ERS-1/2, JERS-1, RADARSAT-1 and Sentinel-1). For airborne SAR such as UAVSAR it can be the [Flight ID](http://uavsar.jpl.nasa.gov/cgi-bin/data.pl) or a similar concept. The center orbit number should be specified if readily available, otherwise the orbit number at the start of the flight can be used instead.
+**sar:platform** is the unique name of the specific platform the instrument is attached to. For satellites this would 
+be the name of the satellite, whereas for drones this would be a unique name for the drone.  It is recommended that 
+the value be lowercase with no spaces. Examples include `sentinel-1a` (Sentinel-1) and `envisat` (Envisat).
+ 
+**sar:constellation** is the name of a logical collection one or more platforms that have similar payloads and have 
+their orbits arranged in a way to increase the temporal resolution of acquisitions of data with similar geometric and 
+radiometric characteristics. This field allows users to search for related data sets without needing to specify from 
+which specific platform the data came. It is recommended that the value be lowercase with no spaces. Most existing 
+SAR systems are single-satellite, so the constellation name is the same as the satellite name, e.g., 
+`sentinel-1a` (Sentinel-1) and `envisat` (Envisat).
```

Comment:
Wait, Sentinel 1 constists of 1A, 1B (at the moment), 1C and 1D (in the future)?!